lib/fluent/plugin/out_file.rb

```diff
@@ -33,6 +37,7 @@ class FileOutput < TimeSlicedOutput
       end
       c
     end
+    desc "A parameter is used for pointing to latest written file."
     config_param :symlink_path, :string, :default => nil
```

Comment:
Should be like

    Create symlink to temporary buffered file when buffer_type is file.